cohort-engine/src/main/java/com/ibm/cohort/engine/translation/CqlTranslationProvider.java

```diff
@@ -19,10 +22,26 @@
  * implementations of a CqlTranslator such as linking directly to the translator
  * JAR or calling out to the CqlTranslationService microservice.
  */
-public interface CqlTranslationProvider {
+public interface CqlTranslationProvider<T> {
 	public Library translate(InputStream cql) throws Exception;
 
 	public Library translate(InputStream cql, List<Options> options) throws Exception;
 
 	public Library translate(InputStream cql, List<Options> options, LibraryFormat targetFormat) throws Exception;
+	
+	public default void registerModelInfo(ModelInfo modelInfo) {
+		// Force mapping  to FHIR 4.0.1. Consider supporting different versions in the future.
+		// Possibly add support for auto-loading model info files.
+		modelInfo.setTargetVersion("4.0.1");
+		modelInfo.setTargetUrl("http://hl7.org/fhir");
+		org.hl7.elm.r1.VersionedIdentifier modelId = (new org.hl7.elm.r1.VersionedIdentifier()).withId(modelInfo.getName()).withVersion(modelInfo.getVersion());
+		ModelInfoProvider modelProvider = () -> modelInfo;
+		ModelInfoLoader.registerModelInfoProvider(modelId, modelProvider);
```

Comment:
Would like this implementation to be in the InJVMTranslationProvider, not the interface.